.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
```

Comment:
You can fully rely on ./run-tests.sh now, though we probably need to add linting into the tests.